### PR TITLE
Cancel FIO request

### PR DESCRIPTION
**Pull request by andreyvEze**

### CODE REVIEW

File: src/components/scenes/FioRequestListScene.js [hunk 1]
```diff
@@ -186,61 +219,98 @@ export class FioRequestList extends React.Component<Props, LocalState> {
...
+
+    if (fioWallet) {
+      try {
+        await cancelFioRequest(fioWallet, parseInt(request.fio_request_id), payeeFioAddress)
+        this.removeFioSentRequest(request.fio_request_id)
+        this.closeRow(rowMap, rowKey)
+        showToast(s.strings.fio_cancel_status)
+      } catch (e) {
+        this.setState({ fullScreenLoader: false })
+        if (e.code && e.code === FIO_NO_BUNDLED_ERR_CODE) {
```
Comment (andreyvEze) [inline src/components/scenes/FioRequestListScene.js:264]: remove `e.code`

File: src/components/scenes/FioRequestListScene.js [hunk 2]
```diff
@@ -420,12 +517,28 @@ export class FioRequestList extends React.Component<Props, LocalState> {
     )
   }
 
+  renderSentHiddenItem = (rowObj: { item: FioRequest }, rowMap: { [string]: SwipeRow }) => {
+    if (isSentFioRequest(rowObj.item.status) || isRejectedFioRequest(rowObj.item.status)) {
+      return null
+    }
+    return (
+      <View style={styles.rowBack}>
```
Comment (andreyvEze) [inline src/components/scenes/FioRequestListScene.js:525]: create squared hidden menu button, props: { primaryText, secondaryText, children }

File: src/modules/FioAddress/util.js [hunk 3]
```diff
@@ -682,3 +682,26 @@ export const getTransferFee = async (fioWallet: EdgeCurrencyWallet | null, forDo
...
   throw new Error(s.strings.fio_get_fee_err_msg)
 }
+
+export const cancelFioRequest = async (fioWallet: EdgeCurrencyWallet | null, fioRequestId: number, fioAddress: string) => {
+  if (!fioWallet) throw new Error(s.strings.fio_wallet_missing_for_fio_address)
+  let getFeeResult
+  try {
+    getFeeResult = await fioWallet.otherMethods.fioAction('getFeeForCancelFundsRequest', { fioAddress })
+  } catch (e) {
+    console.log('=================', e.message)
```
Comment (andreyvEze) [inline src/modules/FioAddress/util.js:692]: remove

Comment (andreyvEze) [reply src/modules/FioAddress/util.js:692]: console.log(e)

**Review by thehobbit85 — APPROVED** [review]

**Pull request merged**